.github/actions/setup/directories/action.yml

```diff
@@ -44,6 +44,11 @@ inputs:
     description: >-
       If set to true, creates dummy files in build dir.
 
+  fetch-depth:
+    required: false
+    default: ''
```

Comment:
Should we set this default value to `1` same as https://github.com/actions/checkout/blob/main/action.yml#L72?

Reply:
Setting the default value `1` is not mandatory because I confirmed the default value is configured correctly when `fetch-depth` is not configured at https://github.com/ruby/ruby/actions/runs/8047076163/job/21975601001?pr=10097#step:4:92. However, I think it would be more readable if we set the default value `1` through this review. Let me change it. Thanks!

Reply:
Updated

Reply:
I see. It's not strong opinion because I'm not sure what happend to pass empty value into `actions/checkout`.